README.md

```diff
@@ -84,16 +87,16 @@ If you are looking to setup YoutubeDL-Material with Docker, this section is for
 
 1. Run `curl -L https://github.com/Tzahi12345/YoutubeDL-Material/releases/latest/download/docker-compose.yml -o docker-compose.yml` to download the latest Docker Compose, or go to the [releases](https://github.com/Tzahi12345/YoutubeDL-Material/releases/) page to grab the version you'd like.
 2. Run `docker-compose pull`. This will download the official YoutubeDL-Material docker image.
-3. Run `docker-compose up` to start it up. If successful, it should say "HTTP(S): Started on port 8998" or something similar.
-4. Make sure you can connect to the specified URL + port, and if so, you are done!
+3. Run `docker-compose up` to start it up. If successful, it should say "HTTP(S): Started on port 8998" or something similar. This tells you the *container-internal* port of the application. Please check your `docker-compose.yml` file for the *external* port. If you downloaded the file as described above, it defaults to **8998**.
```

Comment:
This is my fault for the inaccuracy here, but if you can change it from "HTTP(S): Started on port 8998" to "HTTP(S): Started on port 17443" since that's what actually comes up when you start the container.

But nice catch overall, I did not see this issue before!

Reply:
Yeah that confused me at first - I added this change as well:)